includes/class-wp-job-manager-data-exporter.php

```diff
@@ -68,13 +68,13 @@ public static function user_data_exporter( $email_address ) {
 					}
 				}
 
-				$user_data_to_export[ $name ] = array(
-					'name'  => __( 'Label', 'wp-job-manager' ),
+				$user_data_to_export[] = array(
+					'name'  => $name,
 					'value' => $user_meta,
 				);
 			}
 
-			$export_items = array(
+			$export_items[] = array(
```

Comment:
Looks good! I think the last thing is to set up this array (`$export_items = array();`) before adding an item to it.

Reply:
So always declare variables before using them?